cves/CVE-2012-3442.yml

```diff
@@ -49,7 +49,15 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  The login() and logout() views provided in Django's authentication framework
+  make use of the common "POST-redirect-GET" pattern; a configurable 
+  querystring parameter can be used to specify the location to redirect to on
+  successful submission. Currently, those views perform basic validation to ensure
```

Comment:
I am wary of using "Currently" on a fixed CVE from 8 years ago. It creates ambiguity as to the time period you are intending to reference.